e2e/app/component/runtime-panel.ts

```diff
@@ -199,8 +199,6 @@ export default class RuntimePanel extends BaseHelpSidebar {
     // Runtime panel automatically close after click Create button.
     // Reopen panel in order to check icon status.
     await this.open();
-    // Runtime state transition: Stopping -> None
-    await this.waitForStartStopIconState(StartStopIconState.Stopping, 2 * 60 * 1000);
```

Comment:
so delete can happens real fast. runtime status Stopping icon never shows up in UI.